[PL-51660] Update audit trail (#6946)
harness/developer-hub: docs/platform/governance/audit-trail/audit-trail.md

With Audit Trail in Harness, you can view and track changes to your Harness resources within your Harness account.

> The audit data retention period is 2 years. Harness reserves the right to delete audit data after 2 years. You can request a longer retention period by contacting Harness. For example, if you require audit data for legal discoveries, contact Harness and we can help.

Author: ```suggestion
The audit data retention period is 2 years. Harness reserves the right to delete audit data after 2 years. You can request a longer retention period by contacting Harness. For example, if you require audit data for legal discoveries, contact Harness, and we can help.
```

This topic shows you how to view audit trails for your Harness account.

### Before you begin

* [Learn Harness' Key Concepts](/docs/platform/get-started/key-concepts.md)
* [RBAC in Harness](/docs/platform/role-based-access-control/rbac-in-harness)

### View an audit trail

You can view audit trail data at the Account or Org scope. This topic explains how to view audit trail data at the account scope.

:::info note
The events in the scope of accounts will not be displayed if you are viewing the audit trail at the Org.
:::

In Harness, go to **Account Settings**, and then select **Audit Trail**.

The **Audit Trail** page opens, displaying a record for each event that changed the setup of your Harness account, modules, or Harness entities. By default, Harness shows audit logs for the previous 7 days.

![](../../governance/audit-trail/static/audit-trail-00.png)

It might take a few minutes for events to appear on the Audit Trail. Wait a minute and then refresh your browser if you don't notice an event right away.

For each event record, this view shows the:

* Date and time (**Time**).
* The user who made the change (**User**).

> * **Action** by the user like created, updated, deleted, and so on.

Reviewer: ```suggestion
* **Action** by the user such as created, updated, deleted, and so on.
```

   :::info note
   The Resource Identifier operates in conjunction with the Resource Type. It allows you to use the resource identifier to filter audit events related to a specific resource using that identifier.
   :::

4. In **Filter Name**, enter a name for your filter.

> 5. In **Who can view and edit the filter?** select **Only me** or **Everyone** based on the visibility you want to set for this filter.

Author: ```suggestion
5. Under **Who can view and edit the filter?**, select **Only me** or **Everyone** based on the visibility you want to set for this filter.
```
